src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataInfo.cs

```diff
@@ -32,6 +34,18 @@ public ViewDataInfo(object container, PropertyInfo propertyInfo, Func<object> va
             _valueAccessor = valueAccessor;
```

Comment:
@Eilon would you prefer the old `ViewDataInfo(object container, PropertyInfo propertyInfo, Func<object> valueAccessor)` constructor be `[Obsolete]`d? Not used at all anymore.